### PR TITLE
Feature/2018 01 20 jworks tech blog

### PR DESCRIPTION
**Pull request by andreasevers**

### CODE REVIEW

**Comment by andreasevers** [comment]
Need to add a page with iframe containing:
https://radar.thoughtworks.com/?sheetId=https%3A%2F%2Fdocs.google.com%2Fspreadsheets%2Fd%2F1RmUMYDS70zXAq2HHEzQdqAnrof-2JxptUNY3_hTZcN0

**Comment by MrJean** [comment]
@andreasevers I have pimped your page. Will you also add it in the navigation?

File: _posts/2018-01-20-JWorks-Tech-Blog.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+The Radar is a document that sets out the changes that we think are currently interesting in software development - things in motion that we think you should pay attention to and consider using in your projects.
+It reflects the idiosyncratic opinion of a bunch of senior technologists and is based on our day-to-day work and experiences.
+While we think this is interesting, it shouldn’t be taken as a deep market analysis.
+
+## Quadrants
+
+The quadrants are a categorization of the type of blips:
+
+- **Programming Languages and Frameworks.** Quite straight-forward, languages and software frameworks.
+- **Tools.** These can be components, such as databases, software development tools, such as versions control systems; or more generic categories of tools, such as the notion of polyglot persistence.
```
Comment (MrJean) [inline _posts/2018-01-20-JWorks-Tech-Blog.md]: @andreasevers Version Control Systems?

File: _posts/2018-01-20-JWorks-Tech-Blog.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+- The **Assess** ring are things that you should look at closely, but not necessarily trial yet - unless you think they would be a particularly good fit for you. Typically, blips in the Assess ring are things that we're currently trialling, on our projects.
+- The **Hold** ring is for things that are getting attention in the industry, but we don't think are ready for use. Sometimes this is because we don't think they're mature enough yet: sometimes it means we think they're irredeemably flawed. We don't have an "avoid" ring, but we do throw things in the Hold ring that we wish our clients wouldn't use.
+
+Unlike the quadrants, we do have some quite passionate arguments about which ring a blip should go into. We don't tend to have angry arguments, but rings are what generate the most energetic discussions. Over the course of making the Radar we’ve come up with some useful rules of thumb to help us put things into rings.
```
Comment (MrJean) [inline _posts/2018-01-20-JWorks-Tech-Blog.md]: @andreasevers angry arguments?

**Comment by andreasevers** [comment]
Closes #212

**Pull request merged**